WeakAurasOptions/AuthorOptions.lua

```diff
@@ -2474,10 +2474,12 @@ local function valuesAreEqual(t1, t2)
   if ty1 ~= ty2 then
     return false
   end
+  if ty1 == "number" then
+    return tostring(t1) == tostring(t2)
```

Comment:
Did you consider checking with a small epsilon instead? Seems a bit more approriate to me.

The other change did work in my testing.

Reply:
I did consider it, but i wanted to be clever and not introduce a hidden limitation. I guess checking if they're off by less than 1 in a billion should be fine though.